tools/pythonpkg/src/include/duckdb_python/pandas_type.hpp

```diff
@@ -24,6 +24,9 @@ class PolarsDataFrame : public py::object {
 	using py::object::object;
 
 public:
+	static bool check_(const py::handle &object) {
+		return !object.is_none();
```

Comment:
Mentioned this on the previous pr, but this should use a py::isinstance call instead of this (was originally my mistake in the pandas DataFrame type)